packages/yew/src/dom_bundle/braw.rs

```diff
@@ -0,0 +1,193 @@
+use wasm_bindgen::JsCast;
+use web_sys::Element;
+
+use crate::dom_bundle::bnode::BNode;
+use crate::dom_bundle::traits::{Reconcilable, ReconcileTarget};
+use crate::dom_bundle::utils::insert_node;
+use crate::dom_bundle::BSubtree;
+use crate::html::AnyScope;
+use crate::virtual_dom::VRaw;
+use crate::{AttrValue, NodeRef};
+
+pub struct BRaw {
+    pub html: AttrValue,
+    reference: NodeRef,
+}
+
+impl BRaw {
+    fn create_elements(html: &str) -> Vec<Element> {
+        let div = gloo::utils::document().create_element("div").unwrap();
+        let html = html.trim();
+        div.set_inner_html(html);
+        let children = div.children();
+        let children = js_sys::Array::from(&children);
+        let children = children.to_vec();
+        children
+            .into_iter()
+            .map(|it| it.unchecked_into())
+            .collect::<Vec<_>>()
+    }
+
+    fn detach_bundle(&self, parent: &Element) {
```

Comment:
This only detaches the first child, not all of them. How expensive is it to reuse `VList` here and put them as a `VList` with `VNode` children and keep a `BList` internally that does the diffing, detaching and attaching?

Reply:
As expensive as introducing a dependency on html parser. We need to create parse the string to be able to create a BList from it